### PR TITLE
[OSL] Fix nightly configuration

### PR DESCRIPTION
**Pull request by lampajr**
**Thank you for submitting this pull request**

**referenced Pull Requests**: 
- https://github.com/kiegroup/kie-jenkins-scripts/pull/1320

Fixes:
- Swap NEXT and CURRENT params definition (for OSL): NEXT is `main` and CURRENT is `1.27.x`
- Set drools version to 2.28.0 for `main`

<pre>
How to retest a PR or trigger a specific build:

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
</pre>

### CODE REVIEW

**Review by mareknovotny — APPROVED** [review]

**Review by radtriste — APPROVED** [review]

File: job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy [hunk 1]
```diff
@@ -30,17 +30,17 @@ def KOGITO_BLUE_NEXT_PRODUCT_VERSION='1.13.2.blue'
...
-def SERVERLESS_LOGIC_CURRENT_PRODUCT_CONFIG_BRANCH="master"
-
-def SERVERLESS_LOGIC_NEXT_PRODUCT_VERSION='1.25.0'
-def SERVERLESS_LOGIC_KOGITO_NEXT_PRODUCT_VERSION='1.27.0'
-def SERVERLESS_LOGIC_DROOLS_NEXT_PRODUCT_VERSION='8.27.0'
-def SERVERLESS_LOGIC_NEXT_PRODUCT_BRANCH='1.27.x'
-def SERVERLESS_LOGIC_NEXT_PRODUCT_CONFIG_BRANCH="openshift-serverless-logic/1.27.x"
+def SERVERLESS_LOGIC_NEXT_PRODUCT_VERSION='2.0.0'
+def SERVERLESS_LOGIC_KOGITO_NEXT_PRODUCT_VERSION='2.0.0'
+def SERVERLESS_LOGIC_DROOLS_NEXT_PRODUCT_VERSION='8.28.0'
```
Comment (radtriste) [inline job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy:35]: @lampajr We could ask the `prepare-release-branch` from Kogito project to update that value automatically after creating the branch. what do you think ?
That way, you don't need to keep up on every community release.

Comment (lampajr) [reply job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy:35]: @radtriste I definetely agree with you

**Review by winklerm — APPROVED** [review]
Thanks!

**Comment by radtriste** [comment]
merged so it gets included in the next run in an hour

**Pull request merged**